msal/src/main/java/com/microsoft/identity/client/internal/controllers/BrokerAccountManagerStrategy.java

```diff
@@ -237,6 +246,14 @@ AcquireTokenResult acquireTokenSilent(final AcquireTokenSilentOperationParameter
                         e
                 );
 
+                Telemetry.emit(
+                        new BrokerEndEvent()
+                                .putAction(methodName)
+                                .isSuccessful(false)
+                                .putErrorCode(ErrorStrings.BROKER_REQUEST_CANCELLED)
+                                .putErrorDescription("OperationCanceledException thrown when talking to account manager. The broker request cancelled.")
```

Comment:
nit: create a string const for redundant err messages. Here and below.

Reply:
Cool, thanks! Addressed. :)